pyirf/interpolation.py

```diff
@@ -0,0 +1,190 @@
+"""Functions for performing interpolation of IRF to the values read from the data"""
+
+import numpy as np
+import astropy.units as u
+from astropy.table import Table
+from astropy.io import fits
+from scipy.interpolate import griddata
+
+
+def interpolate_effective_area_per_energy_and_fov(effective_area, grid_points, target_point, min_effective_area=1. * u.Unit('m2'), method='linear'):
+    """
+    Takes a grid of effective areas for a bunch of different parameters
+    and interpolates (log) effective areas to given value of those parameters
+
+    Parameters
+    ----------
+    effective_area: np.array of astropy.units.Quantity[area]
+        grid of effective area, of shape (n_grid_points, n_fov_offset_bins, n_energy_bins)
+    grid_points: np.array
+        list of parameters corresponding to effective_area, of shape (n_grid_points, n_interp_dim)
+    target_point: np.array
+        values of parameters for which the interpolation is performed, of shape (n_interp_dim)
+    min_effective_area: astropy.units.Quantity[area]
+        Minimum value of effective area to be considered for interpolation
+    method: 'linear’, ‘nearest’, ‘cubic’
+        Interpolation method
+
+    Returns
+    -------
+    aeff_interp: astropy.units.Quantity[area]
+        Interpolated Effective area array with shape (n_energy_bins, n_fov_offset_bins)
+    """
+
+    _, n_fov_offset_bins, n_energy_bins = effective_area.shape
+
+    # get rid of units
+    effective_area = effective_area.to_value(u.m**2)
+    min_effective_area = min_effective_area.to('m2').value
```

Comment:
`to_value(u.m**2)` here as well

Reply:
done,

do you know if there is an actual change in performance between to(...).value and to_value(...), or is it just a question of style?